x/evmutil/keeper/erc20.go

```diff
@@ -19,34 +19,32 @@ const (
 	erc20BalanceOfMethod = "balanceOf"
 )
 
-// DeployTestMintableERC20Contract deploys an ERC20 contract on the EVM as the
+// DeployTestERC20Contract deploys an ERC20 contract on the EVM as the
```

Comment:
curious why "Test" is in the function name?

Reply:
Refactored in https://github.com/Kava-Labs/kava/pull/1448